### PR TITLE
update sdk version

### PR DESCRIPTION
**Pull request by leyou240**

### CODE REVIEW

**Comment by YangruiEmma** [comment]
please fix the ci cases first, thx

**Comment by leyou240** [comment]
> please fix the ci cases first, thx

I just change v2 dir, and works fine.

`Error: go/pkg/mod/github.com/apache/thrift@v0.13.0/lib/go/test/tests/protocol_mock.go:27:2: package thrift is not in GOROOT `

I have no idea about it in v1 dir~

File: go.mod [hunk 1]
```diff
@@ -1,12 +1,63 @@
 module github.com/kitex-contrib/registry-nacos
 
-go 1.16
+go 1.18
 
 require (
-	github.com/apache/thrift v0.13.0
-	github.com/cloudwego/kitex v0.5.1
+	github.com/apache/thrift v0.16.0
```
Comment (YangruiEmma) [inline go.mod]: should not upgrade thrift, apache thrift has breaking change

Comment (leyou240) [reply go.mod]: `
replace github.com/apache/thrift => github.com/apache/thrift v0.13.0
`
so it does not matter.

**Comment by leyou240** [comment]
> please fix the ci cases first, thx

done @YangruiEmma

File: v2/registry/registry.go [hunk 2]
```diff
@@ -21,7 +21,9 @@ import (
 	"strconv"
 
 	"github.com/cloudwego/kitex/pkg/registry"
+
```
Comment (baiyutang) [inline v2/registry/registry.go]: `gofumpt -w -l .`
先解决下格式问题

Comment (leyou240) [reply v2/registry/registry.go]: done

**Review by baiyutang — APPROVED** [review]

**Pull request merged**